pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArrayFunctions.java

```diff
@@ -74,6 +76,85 @@ public static int arrayIndexOfString(String[] values, String valueToFind) {
     return ArrayUtils.indexOf(values, valueToFind);
   }
 
+  @ScalarFunction
+  public static int[] arrayIndexOfAllInt(int[] value, int valueToFind) {
```

Comment:
Consider naming it `arrayIndexesOfInt()` (align with the method name in `ArrayUtils`), same for other methods